Resources/Prototypes/Entities/Structures/Walls/walls.yml

```diff
@@ -640,10 +645,11 @@
     key: walls
     base: silver
 
+#shuttle walls
 - type: entity
   id: WallShuttleDiagonal
   name: shuttle wall
-  suffix: diagonal
+  suffix: Diagonal
```

Comment:
should be lowercase still.

Reply:
All the others are upper-case. So, this was for consistency.